### ISSUE TITLE
[Feature request]: Shortcut to take a screenshot

### ISSUE THREAD

**Issue by Storyyeller** [visualboyadvance-m/visualboyadvance-m#1145]
**Update**: I figured out a way to do this. The trick is to go into input configuration, switch to the "special" tab and assign a key to "Screenshot" rather than using "Screen capture" on the main menu. I wish this were more discoverable - I only figured it out by digging through the source code.

### Request box

I wish there were a way to take a screenshot with just one keypress. Currently, the "Screen capture" option only pops up a file save dialog and forces you to switch to the dialog and type in a new filename and then hit save, *every single time you take a screenshot*, which is infeasible if you want to take a large number of screenshots during gameplay.

There should be a shortcut which will *automatically* take a screenshot and save it with a unique filename to the screenshots directory **without prompting the user** or forcing them to leave the game.

### Reason box

Having to manually name and save every single screenshot makes it impossible to take large numbers of screenshots during gameplay and is very inconvenient in general.

### Examples

Not sure, haven't tried other emulators

**Comment by skeddles** [comment]
yeah many other emulators just save the screenshots automatically, you dont have to pick a location. i agree this makes it impossible to take lot of screenshots

**Comment by Squall-Leonhart** [comment]
the screen capture button is not actually for screenshots but making game covers for folders and front ends,  not sure why the screenshot button isn't assigned to printscreen by default
